python/pyspark/pandas/sql_processor.py

```diff
@@ -65,6 +66,9 @@ def sql(
     ----------
     query : str
         the SQL query
+    index_col : str or list of str, optional
```

Comment:
I mean I don't know the standard way of describing the list of strings (`list of str`). Are you able to find any example like this in pandas or NumPy documentation?